securedrop/source_templates/index.html

```diff
@@ -34,15 +34,16 @@
           </h2>
           <hr class="cut-out" />
           <p>If this is your first time submitting documents to journalists, start here.</p>
-          <a href="/generate" class="btn alt block" id="submit-documents-button"><img id="warning-close" src="{{ url_for('static', filename='i/font-awesome/white/cloud-upload.svg') }}" width="17px" height="17px"> Submit Documents</a>
+          {# adding a break between 'submit' and 'documents' to make both containers equal sized #}
+          <a href="/generate" class="sd-button btn alt block" id="submit-documents-button">SUBMIT<br/>DOCUMENTS</a>
```

Comment:
Ok branched off this and in `fix-box-sizes-on-src-interface-index` there is a single commit 74f612b4ef42679d1f6f05f223f34700cbccd254 fixing these boxes to be the same size. Now in Tor Browser this looks like:

![modified](https://cloud.githubusercontent.com/assets/7832803/22532860/09fb5fd0-e89f-11e6-8c89-80889446d703.png)

I'll submit a PR against this repo to get that commit merged in after I merge this branch in